src/components/PetCard.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="pet-selection" @click="selectPet">
+    <div class="pet-card">
+      <img :src="pet.image" :alt="pet.name">
+      <div class="pet-card-name">
+        <p>{{ pet.name }}</p>
+        <img src="../assets/icon_audio.svg" alt="Audio">
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import type { Pet } from '@/config/petsConfig';
+import { useRouter } from 'vue-router';
+
+const props = defineProps<{
+  pet: Pet;
+}>();
+
+const router = useRouter();
+
+const selectPet = () => {
```

Comment:
There is no need for this function, you can just call `handlePetSelected `directly.
`<div class="pet-selection" @click="handlePetSelected">`